src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/SampleData.java

```diff
@@ -120,10 +120,10 @@ public static String fileContentAsString(String file) {
     }
 
     public static Envelope envelope(int numberOfDocuments) {
-        return envelope(numberOfDocuments, ImmutableMap.of("fieldName1", "value1"));
+        return envelope(numberOfDocuments, ImmutableMap.of("fieldName1", "value1"), false);
     }
 
-    public static Envelope envelope(int numberOfDocuments, Map<String, String> ocrData) {
+    public static Envelope envelope(int numberOfDocuments, Map<String, String> ocrData, boolean subtypeValuesNull) {
```

Comment:
subtypeValuesNull ?

Reply:
I want to set subtype value to null for all documents. this is used for unit tests.